Gopkg.toml

```diff
@@ -16,6 +16,9 @@ required = [
 [prune]
   go-tests = true
 
+[[constraint]]
+  name="sigs.k8s.io/cluster-api"
+  revision="f8522af25c3a1cbcb304cf1034ec8d40d7153a4c"
```

Comment:
Is this necessary to update? Can you just run dep ensure update for this package to update the gopkg.lock instead of inserting a constraint here?